### DISCUSSION TITLE
Disabling "?url=" parameter in swagger

### DISCUSSION THREAD

**Discussion by alegmal** [swagger-api/swagger-ui#7610] [Q&A] [answered]
Hi,

Currently one can use "/?url=" parameter to 'inject' their code into swagger and potentially see confidential information such as cookies / headers that contain private keys. How can I disable / block the use of "/?url"?

**Comment by ponelat** [comment, marked as answer]
Thanks @alegmal, heads up that https://github.com/swagger-api/swagger-ui/issues/4872 will be worked on soon.
